.gitignore

```diff
@@ -1,3 +1,6 @@
 *.rs.bk
 Cargo.lock
 target
+target-docker
+compiler-rt
+*.tar.gz
```

Comment:
I don't think these changes should be committed. Maybe only the compiler-rt one?

Reply:
Ah, `target-docker` was from before I changed it to just use a docker volume. Removed this.

tar.gz is just there so if you download the C libraries like in CI it won't get picked up. Is this okay to keep?